docs/vickywane-ambianic-stripe-swagger-docs-0.1-resolved.yaml

```diff
@@ -0,0 +1,51 @@
+---
+openapi: 3.0.1
+info:
+  title: Ambianic Serverless functions API Definition
+  description: An API Definition for Ambianic Serverless functions created using Swagger
+  version: "0.1"
+servers:
+- url: https://ambianic-functions.netlify.app
+- url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+paths:
+  /create-stripe-subscriber:
+    post:
+      description: Serverless function for creating a customer on stripe and subscribing them to an item. 
+      responses:
+        "200":
+          description: Returns stripe customer response.
+          content:
+            text/html; charset=utf-8:
+              schema:
+                type: string
+              examples: {}
+      servers:
+      - url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+    servers:
+    - url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+  /.netlify/functions/getCustomers:
```

Comment:
@vickywane similar comment here. We end up with a strange auto generated function name such as `netlifyFunctionsGetCustomersGet` because we are starting off with an inappropriate method path `/getCustomers`. The path should be `/customers` and that would make the auto-generated function name look less weird. With `operationId` we can tune up the function name to be self descriptive in case the auto-generated name does not look right.

Reply:
The has been addressed starting with the name of the serverless functions endpoint name.